site/docs/components/_content-template/index.mdx

```diff
@@ -0,0 +1,218 @@
+-->
+
+## When to use Component Name
+
+<!-- Bulleted list of scenarios in which this component can be used  -->
+
+- ...
+- ...
+
+## When not to use Component Name
+
+<!--
+  Bulleted list of scenarios where this component is not appropriate. Where
+  possible, include a link to an alternative component that should be used in
+  it's place
+-->
+
+- ...
+- ...
+
+## Configurations
+
+<!--
+  A visual list of possible configurations of the component.
+
+  The purpose is to convey the main features and/or usage options of the
+  component (e.g. the Data Grid's striped rows might be a "configuration" to
+  showcase here). The configurations listed here should aim to be comprehensive,
+  but not necessarily exhaustive (some complex components may have vast numbers
+  of possible permutations, so listing out each one would be impossible and
+  overwhelming to readers).
+
+  Each confguration should have its own section following the structure outlined
+  below:
+-->
+
+### Configuration title (e.g. Striped rows)
+
+<!--
+  A brief description of what this configuration provides to users and when it
+  should be used.
+-->
+
+<!--
+  A live example of the component configuration, with access to the correspoding
+  source code.
+
+  In Storybook this is done using something like:
+
+  <Canvas>
+    <Story id="..." />
+  </Canvas>
+-->
+
+## Content
+
+<!--
+  OPTIONAL: Remove this section if it is not applicable.
+
+  Add any component-specific content guidance here, use visual examples where
+  appropriate.
+
+  For example, a Button component might include guidance on recommended terms to
+  use for the label text, recommended maximum character counts, etc.
+-->
+
+## Accessibility
+
+<!--
+  OPTIONAL: Remove this section if it is not applicable.
+
+  Describe any considerations the users may need to take into account when using
+  the component.
+-->
+
+## References
+
+<!--
+  OPTIONAL: Remove this section if it is not applicable.
+
+  If there are industry standards for this component, list them in this
+  section. Include links out to useful articles.
+-->
+
+- [Relevant industry standard](https://example-standard.org/)
+- [Useful article](https://some-site.com/article/bla)
+- ...
+
+## Designing
+
+### Theming considerations
+
+<!--
+  Describe how Salt characteristics and theming impact the component, and any
+  modifications that can be made.
+-->
+
+### Design guidance
+
+<!--
+  Add a bulleted list of any other design-related guidance the user may need to
+  consider.
+-->
+
+### Additional materials
+
+<!--
+  Add a bulleted list of links to additional, component-specific materials:
+
+  * [Full component specification (in Figma)](https://figma.com/path/to/spec)
+  * [Figma stickersheet](https://figma.com/path/to/stickersheet)
+  * ...
+-->
+
+## Developing
+
+### Import
+
+<!--
+  Show the typical import statement used to include this component in your
+  code.
+-->
+
+```js
+import { ComponentName } from "@some-org/some-package";
+```
+
```

Comment:
After import section, could we list out a bunch of component specific features here? before the props table, in a bulleted list?

Reply:
I'm not sure what you mean. Can you provide an example of what you have in mind? Also, how would this differ from what's being conveyed in the "Configurations" section?

Reply:
Sure, have tagged you in Figma. Actually now to think about it, it wont differ from the 'configurations' section, it just means when the time comes to rolling out the new template, I move that info somewhere else, instead of having a description and live example for each feature :)

Reply:
Cool. We'll leave this as is for now then.